alphastats/llm/uniprot_utils.py

```diff
@@ -270,6 +291,7 @@ def get_info(genes_list: List[str], organism_id: str) -> List[str]:
 
     for gene in genes_list:
         result = get_uniprot_data(gene, organism_id)
+        result = result["results"][0]
```

Comment:
maybe add this comment `Return the first result as a dictionary (assuming it's the most relevant)` here?

Reply:
I didn't bother because these functions will be depracated once the is completely based on protein ids to feed the LLM. But sure, I can add it back.